docs: add changelog for bridge_v2 (#11851)
emqx/emqx: changes/ee/feat-11581.en.md

Preview Feature: Support for Version 2 Bridge Design

- Introduction of Bridge v2 with a new 'connector' concept

  - In the original Bridge v1 design, each connector was exclusively tied to a single bridge.
    This design prioritized error isolation and performance but posed challenges for users setting up multiple bridges to the same service.
    For instance, setting up 10 bridges to a single Kafka cluster required the same configuration to be repeated for each bridge.

>   - The revamped Bridge v2 design provides more flexibility:

Reviewer: ```suggestion
  - The revamped Bridge v2 design provides more flexibility and better scalability:
```

    - Users have the option to either share a connector across multiple bridges or retain it exclusively for one bridge, as in v1.

>     - For the majority of data bridges, sharing a connector among multiple bridges might lead to performance degrade.

Reviewer 2: ```suggestion
    - For the majority of data bridges, sharing a connector among multiple bridges might lead to performance degradation.
```

Preview Feature: Support for Version 2 Bridge Design

- Introduction of Bridge v2 with a new 'connector' concept

  - In the original Bridge v1 design, each connector was exclusively tied to a single bridge.
    This design prioritized error isolation and performance but posed challenges for users setting up multiple bridges to the same service.
    For instance, setting up 10 bridges to a single Kafka cluster required the same configuration to be repeated for each bridge.

  - The revamped Bridge v2 design provides more flexibility and better scalability:
    - Users have the option to either share a connector across multiple bridges or retain it exclusively for one bridge, as in v1.
    - For the majority of data bridges, sharing a connector among too many bridges might lead to performance degradation but avoids

>       overlading the external system with too many connections if the number of bridges is very high.

Reviewer: ```suggestion
      overloading the external system with too many connections if the number of bridges is very high.
```

>     - For a few data bridges, it is always dedicated connections used even when connector is shared. These bridges are:

Reviewer: ChatGPT suggests:

```suggestion
    - In some cases, specific data bridges always utilize dedicated connections, even when the connector is shared. Right now these are:
```
